api/shape/Plan.js

```diff
@@ -211,6 +211,52 @@ export const getScale = (geometry) => {
   ];
 };
 
+const X = 0;
+const Y = 1;
+const Z = 2;
+
+export const buildCorners = (x, y, z) => {
```

Comment:
This is a neat idea!

Reply:
Yeah, should have factored that out earlier. :)